style.css

```diff
@@ -0,0 +1,284 @@
+* {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+}
+
+body {
+    margin:0 auto;
+    height: 100%;
+}
+
+nav {
+    margin: 0;
+    padding: 0;
+    background-color: #111111;
+    width: 100%;
+    height:auto;
+    position: fixed;
+    top: 0;
+    color: #dddddd;
+    font-family: 'Lato', sans-serif;
+    z-index: 5;
+}
+
+.navwrapper {
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    max-width: 65%;
+    margin:0 auto;
+    -webkit-box-pack: justify;
+    -ms-flex-pack: justify;
+    justify-content: space-between;
+}
+
+.navitem {
+    margin: 0;
+    padding: 0 20px;
+    -ms-flex: 0 1 auto
+}
+
+.navitem h4 {
+    font-size: 1.4rem;
+    margin: 10px 0 2px 0;
+}
+
+.navitem p {
+    font-size: 0.6rem;
+    margin: 2px 0 10px 0;
+}
+
+header {
+    margin: 61.5px 0 0 0;
+    padding: 0;
+    background-color: #eeeeee;
+    height: auto;
+    overflow: hidden;
+}
+
+.headerwrapper {
+    max-width: 65%;
+    height: 100%;
+    margin: 0 auto;
+    background-size: auto 100%;
+    background-image: url('headerbackground.png');
+    background-repeat: no-repeat;
+    background-position: right;
```

Comment:
worth changing this to left or centre on smaller screen sizes so you can see more of the image

Reply:
changed in 28201fb